x-pack/legacy/plugins/siem/public/containers/case/api.ts

```diff
@@ -72,3 +82,27 @@ export const updateCaseProperty = async (
   await throwIfNotOk(response.response);
   return convertToCamelCase<Partial<CaseSnake>, Partial<Case>>(response.body!);
 };
+
+export const createComment = async (newComment: NewComment, caseId: string): Promise<Comment> => {
```

Comment:
nit -> we should be able to use the query attribute from fetch options.

Reply:
I added a task for each of us ([API](https://github.com/elastic/kibana/issues/58525) and [View](https://github.com/elastic/kibana/issues/58526)) to get this done